packages/docsearch-css/src/modal.css

```diff
@@ -597,9 +602,9 @@ assistive tech users */
   .DocSearch-Modal {
     border-radius: 0;
     box-shadow: none;
-    height: 100vh;
-    height: -webkit-fill-available;
+    height: max-content;
```

Comment:
Does any browser support `max-content` but not `calc()`? I doubt it, so I would remove this line as it will always be overridden by the next one.